src/web/mjs/connectors/ReadComicBooksOnline.mjs

```diff
@@ -61,20 +61,11 @@ export default class ReadComicBooksOnline extends Connector {
     /**
      *
      */
-    _getPageList( manga, chapter, callback ) {
-        let request = new Request( this.url + chapter.id, this.requestOptions );
-        this.fetchDOM( request, 'source.picture' )
-            .then( data => {
-                let pageList = data.map( element => this.createConnectorURI( {
-                    url: this.getAbsolutePath( '/reader/' + element.getAttribute( 'src' ), request.url ),
-                    referer: request.url
-                } ) );
-                callback( null, pageList );
-            } )
-            .catch( error => {
-                console.error( error, chapter );
-                callback( error, undefined );
-            } );
+    async _getPages(chapter) {
+        let request = new Request(new URL(chapter.id, this.url), this.requestOptions);
+        let data = await this.fetchRegex(request, /messages\s*=\s*(\[.*\])\s*;/g);
```

Comment:
I would change the regex to be either non-greedy or consume all characters except `]` in the capture group, otherwise it may include all `]` up to the last closing `]` which is probably not desired
`/messages\s*=\s*(\[[^\]]+\])\s*;/g`
See: https://regex101.com/r/M7ZepM/1